### DISCUSSION TITLE
Doxygen Elements

### DISCUSSION THREAD

**Discussion by tinygiant98** [tree-sitter/tree-sitter#1848] [Grammars] [answered]
After a bit of searching, I was unable to find a source for grammar definitions for doxygen under tree-sitter.  Does any know if there are grammar elements for finding doxygen comments within c comments already written?  Thanks!

**Comment by ObserverOfTime** [comment, marked as answer]
https://github.com/tree-sitter-grammars/tree-sitter-doxygen
